sdk/core/Azure.Core/src/AsyncPageable.cs

```diff
@@ -128,7 +128,7 @@ public static AsyncPageable<T> FromPages(IEnumerable<Page<T>> pages)
         /// A string representation of an <see cref="AsyncPageable{T}"/>.
         /// </returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public override string ToString() => base.ToString();
+        public override string? ToString() => base.ToString();
```

Comment:
When the BCL enabled nullable annotations they've marked Object.ToString as returning a nullable string we are only flowing the value through without changes so have to stick to the same semantics.

This became a problem only now because when targeting netstandard2.0 and net471 we were not compiling against libraries that have nullable annotation (they were added in netcoreapp 3 and we never targeted that). On the other hand in .net 5 these annotations are available and the compiler produces an error saying that `base.ToString()` returns a nullable string and our method was defined as returning a non-nullable. That's why it had to be adjusted.